src/assets/source/sass/components/_theme.scss

```diff
@@ -1,7 +1,8 @@
-body {
+html, body {
   background-color: #EEF1F4;
   background-image: url('~/img/background.jpg');
-  font-family: 'Lato', sans-serif;
+  font-family: "Raleway", sans-serif;
```

Comment:
I think there is a variable in Bootstrap which you can override, use for declaring default font.